app/api/__init__.py

```diff
@@ -97,19 +97,19 @@
 # microlocations
 api.route(MicrolocationList, 'microlocation_list', '/microlocations', '/sessions/<int:id>/microlocations')
 api.route(MicrolocationDetail, 'microlocation_detail', '/microlocations/<int:id>',
-          '/events/<int:event_id>/microlocations', '/events/<identifier>/microlocations',
-          '/sessions/<int:session_id>/microlocation')
+          '/events/<int:event_id>/microlocations', '/sessions/<int:session_id>/microlocation')
 api.route(MicrolocationRelationship, 'microlocation_session',
           '/microlocations/<int:id>/relationships/sessions')
 api.route(MicrolocationRelationship, 'microlocation_event',
           '/microlocations/<int:id>/relationships/event')
 
 # sessions
-api.route(SessionList, 'session_list', '/sessions', '/events/<int:event_id>/sessions', '/events/<identifier>/sessions',
+api.route(SessionList, 'session_list', '/sessions', '/events/<int:event_id>/sessions', '/events/<event_identifier>/sessions',
           '/tracks/<int:track_id>/sessions', '/session_types/<int:session_type_id>/sessions',
           '/microlocations/<int:microlocation_id>/sessions')
 api.route(SessionDetail, 'session_detail', '/sessions/<int:id>',
-          '/microlocations/<int:microlocation_id>/sessions', '/events/<int:event_id>/microlocations')
+          '/microlocations/<int:microlocation_id>/sessions', '/events/<int:event_id>/microlocations',
```

Comment:
Why are `/microlocations/<int:microlocation_id>/sessions`, `/events/<int:event_id>/microlocations` & `/events/<event_identifier>/microlocations` even here ? This is session detail.